### DISCUSSION TITLE
Is this project not being actively maintained anymore?

### DISCUSSION THREAD

**Discussion by stephenleo** [zilliztech/GPTCache#608] [General]
Seems like there isn't any active development in sometime. Wanted to check if this project is not being actively maintained anymore? The main issue is the breaking changes with openai v1 SDK causing a lot of things to stop working.

**Comment by xiaofan-luan** [comment]
we should refine the API in the near future.

Anyone interested please contribute on that

**Reply by stephenleo** [reply]
Thanks! Any suggestions on how to get started?

I see that currently, the `openai.ChatCompletion` class is inherited, and the `create` method is overridden in `adapter/openai.py`
But with openAI API removing the ChatCompletion class and moving to `client.chat.completions.create`, this method doesn't directly work anymore.
